### PR TITLE
Add HSTRLEN command implementation and testing

### PR DESCRIPTION
**Pull request by eitanhs**
implements #1238

### CODE REVIEW

**Comment by eitanhs** [comment]
What version of Redis server exists in the CI environment?
The HSTRLEN command is available since 3.2.0. and the tests are failing due to 'unknown command'...

**Comment by mgravell** [comment]
Looks good to me. Can I confirm that you're free and able to freely contribute the code in the same terms as the project license etc?

**Comment by eitanhs** [comment]
> Looks good to me. Can I confirm that you're free and able to freely contribute the code in the same terms as the project license etc?
Sure!

**Comment by NickCraver** [comment]
Just getting to some of this - apologies for the delay.

We need to add some checks against the version here so that things work, check out the Streams test, specifically: `Skip.IfMissingFeature(conn, nameof(RedisFeatures.Streams), r => r.Streams);` to see how this should be added to features based on version and tests skipped at lower versions :)

**Comment by eitanhs** [comment]
@NickCraver - thanks for the feedback! fixed the relevant test.
Unfortunately the CI is still unhappy, but the failing tests seem unrelated to my changes...

File: tests/StackExchange.Redis.Tests/WrapperBaseTests.cs [hunk 1]
```diff
@@ -130,7 +130,14 @@ public void HashSetAsync_2()
...
             mock.Verify(_ => _.HashSetAsync("prefix:key", "hashField", "value", When.Exists, CommandFlags.None));
         }
-
+        
+        [Fact]
+        public void HashStringLengthAsync()
+        {
+            wrapper.HashStringLengthAsync("key","field", CommandFlags.None);
+            mock.Verify(_ => _.HashStringLengthAsync("prefix:key", "field", CommandFlags.None));
+        }
```
Comment (NickCraver) [inline tests/StackExchange.Redis.Tests/WrapperBaseTests.cs]: Can you please add the equivalent in DatabaseWrapperTests for non-async (you'll see the others mirrored there - e.g. below `HashSet_2`)? Happy to take as a TODO after the merge as well, just let me know :)

Comment (eitanhs) [reply tests/StackExchange.Redis.Tests/WrapperBaseTests.cs]: Done

**Comment by eitanhs** [comment]
I hope I read you correctly and you meant that I'll add another UT for the async version of the method in `Strings.cs` ? If so - done :)

What's weird is that there isn't any UTs for the sync version of any other method beside `HashStringLength` as far as I can see... is that on purpose or am I missing something?

**Comment by NickCraver** [comment]
You're all good - agree there's some gaps there I'll go back and normalize :)

**Comment by eitanhs** [comment]
Awesome, thanks!

**Pull request merged**